yarn.lock

```diff
@@ -21,8 +21,8 @@
     "@aws-amplify/core" "^2.3.0"
     uuid "^3.2.1"
 
-"@aws-amplify/api@^2.2.1":
-  version "2.2.1"
+"@aws-amplify/api@^3.2.16":
```

Comment:
This is a major bump, I will suggest running the demo app to see if all works fine. If all good we can bump this.